docker/README.md

```diff
@@ -0,0 +1,122 @@
+# Celestia `txsim` Docker Image Usage Guide
+
+The `txsim` binary is a tool that can be used to simulate transactions on the Celestia network. It can be used to test the performance of the Celestia network.
```

Comment:
[no change needed][follow up issue] there are other Dockerfiles in this directory. They weren't touched in this PR but maintainers should consider adding to this README what the other files do (i.e. `Dockerfile_ephemeral`)

Reply:
Yes. Ideally, we need to make a dedicated issue to document other 2 DockerFiles we have in the repo. 
I started this ReadMe, so we can spec out not only the protocol, but what options do we have to run the software apart from vanilla `make install && celestia-appd xyz` in the host machine